ci/travis/run_tests.sh

```diff
@@ -16,4 +16,4 @@ export CUNIT_VALGRIND_FLAGS="--leak-check=full --error-exitcode=1"
 
 # Standard build
 ./configure CFLAGS="${CFLAGS_STD}" LDFLAGS="${LDFLAGS_STD}"
-bash ./ci/travis/logbt -- make -j check RUNTESTFLAGS="--verbose --extension"
+bash ./ci/travis/logbt -- make -j installcheck RUNTESTFLAGS="--verbose"
```

Comment:
We want _both_ `make check` and `make installcheck`. The first checks the installation via manual scripts, the second checks via extension

Reply:
Of course the `installcheck` must be done _after_ `make install`